### ISSUE TITLE
windows: docker: command not found( docker not added to path) 

### ISSUE THREAD

**Issue by Rykka** [docker/toolbox#260]
Running with mingw64 with this problem,
Solved with adding 'C:\Program Files\Docker Toolbox' to $PATH

**Comment by fbender** [comment]
Confirm on Win7, Toolbox 1.9.1c

**Comment by dubrousky** [comment]
Confirmed on Win8.1

**Comment by lmmarsano** [comment]
raised here docker/docker#17774

**Comment by bandrzej** [comment]
...and here:
 PATH confusion #469

**Comment by nishidayuya** [comment]
For normal user (non-administrator), confirmed on Windows Server 2008 R2, Docker Toolbox 1.11.1b.

Administrator works fine without PATH setting.

**Comment by amitntrivedi** [comment]
Yes setting up path by right lick on computer icon on desk top and setting environment parameter solve this problem.

**Comment by AndreKR** [comment]
I created a fix for that a long time ago (https://github.com/docker/toolbox/pull/482), but it's not getting merged.

**Comment by jasonmutd** [comment]
yes, adding path to the environment variable solve this issue for windows10. Remember to restart the docker terminal after you added the path.

**Comment by alexandis** [comment]
I have the same issue. Despite having docker directory added to PATH (and it's obvious from the screenshot it's accessible) I cannot install image using bash:
![image](https://user-images.githubusercontent.com/6149843/79585998-3d203180-80d9-11ea-8bda-0ee873d1b436.png)

**Comment by MetSystem** [comment]
The problem is caused by an environment variable. Please keep docker solely for all deletions in the environment variable.

**Comment by devniel** [comment]
Same problem, also by running it with `bash` on powershell.

**Comment by sdktraceur** [comment]
Still exists
